sql/core/src/test/resources/sql-tests/inputs/timestamp.sql

```diff
@@ -102,6 +102,13 @@ select timestamp'2011-11-11 11:11:11' - interval '2' second;
 select '2011-11-11 11:11:11' - interval '2' second;
 select '1' - interval '2' second;
 select 1 - interval '2' second;
+select '2011-11-11 11:11:11' - date'2011-11-11';
```

Comment:
> how about string - date/timestamp?

I have added a new test case. But it seems that `string - date` will be `cast(string as date) - date`.  Should we move it to date.sql or just leave it here?

Reply:
let's put it in `date.sql`, as we don't need to test this query 4 times with different timestamp settings.